src/lib/modules/console/index.ts

```diff
@@ -46,6 +46,10 @@ class Console {
 
     if (this.ipc.isServer()) {
       this.ipc.on("console:executeCmd", this.executeCmd.bind(this));
+      this.ipc.rawApi().server.on("console:history:save", (cmd: string) => {
+        this.events.emit("console:history:save", cmd);
+        this.ipc.rawApi().server.broadcast("console:history:save", cmd);
```

Comment:
Could this not be simplified to `this.ipc.broadcast("console:history:save", cmd)`?

Reply:
embark's `IPC#broadcast` broadcasts the return of `require('flatted/cjs').stringify(data)` such that receivers need to parse the message, which makes sense for the existing use cases of `IPC#broadcast`. In this situation, though, it's overkill — we can just broadcast/receive the command string.

Reply:
As above, could we pass a `raw` parameter to `broadcast` so that the message is not transformed?

Reply:
See https://github.com/embark-framework/embark/pull/1213#discussion_r243480741.